public/src/components/channelManagement/supportLandingPage/copyEditor.tsx

```diff
@@ -0,0 +1,157 @@
+import { SupportLandingPageCopy } from '../../../models/supportLandingPage';
+import React, { useEffect, useState } from 'react';
+import { templateValidatorForPlatform } from '../helpers/validation';
+import { Controller, useForm } from 'react-hook-form';
+import { getRteCopyLength, RichTextEditorSingleLine } from '../richTextEditor/richTextEditor';
+import { makeStyles } from '@mui/styles';
+import { Theme } from '@mui/material';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  container: {
+    marginLeft: spacing(2),
+  },
+}));
+
+const headingCopyRecommendedLength = 500;
+const subheadingCopyRecommendedLength = 500;
```

Comment:
Can you go over over the word count in the client? Or do we prevent that from happening?